scripts/train_ui.py

```diff
@@ -2,7 +2,7 @@
 
 script_imports()
 
-from modules.ui.TrainUI import TrainUI
+from modules.ui.TrainUI import TrainUI  # noqa: E402
```

Comment:
You only added the `# noqa: E402` comment in this script. Is there a difference compared to all the other sctipts?

Reply:
The imports aren't sorted:

```
scripts/train_ui.py:5:1: E402 Module level import not at top of file
```

Reply:
This probably comes from the `script_imports()` call in line 3. But I still don't understand why this isn't an issue in any of the other scripts. For example, generate_masks.py starts like this:
```python
from util.import_util import script_imports

script_imports()

from modules.module.ClipSegModel import ClipSegModel
```
and there is no #noqa comment in that file.

Reply:
I've disabled this lint.